packages/dev/docs/src/docs.js

```diff
@@ -46,6 +50,40 @@ for (let link of links) {
   link.parentNode.replaceChild(container, link);
 }
 
+function ImageModal({children}) {
+  let [trigger, contents] = React.Children.toArray(children);
+  let [isOpen, setOpen] = React.useState(false);
+  let {pressProps} = usePress({
+    onPress: () => setOpen(true)
+  });
+
+  trigger = React.cloneElement(trigger, pressProps);
+  return (
+    <>
+      {trigger}
+      <Modal isDismissable isOpen={isOpen} onClose={() => setOpen(false)} UNSAFE_style={{overflow: 'scroll'}}>
+        {contents}
+      </Modal>
+    </>
+  );
+}
+
+for (let image of images) {
+  let container = document.createElement('span');
+  let url = image.src.replace(/.*\/\/[^/]*/, '');
+
+  ReactDOM.render(
+    <ThemeProvider UNSAFE_className={docsStyle.inlineProvider}>
+      <ImageModal>
+        <img src={url} className={image.className} alt={image.alt} />
```

Comment:
actually, that probably goes here, sorry

Reply:
We should probably allow Esc to close it. Also, maybe clicking the popup also closes it? unsure on that one. Definitely ESC to close though

Reply:
Sounds good, thanks for catching the Esc case, I though Modal would handle it but it doesn't. I'll probably change the trigger to be `<input type="image" />`.  As for `aria-controls`/`aria-haspopup`, I checked the 1.2 aria example of a modal dialog and it didn't have those on the trigger: https://w3c.github.io/aria-practices/examples/dialog-modal/dialog.html